superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -54,7 +54,7 @@ def format(self, article, subscriber):
 
     def _transform_to_ninjs(self, article, recursive=True):
         ninjs = {
-            '_id': article[GUID_FIELD],
+            'guid': article[GUID_FIELD],
```

Comment:
not sure this will work, seems like in http push service `guid` is set using `_id` value and `_id` is removed anyhow